src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -241,4 +247,25 @@ public void resetPosition() {
   private double squareInput(double value) {
     return Math.copySign(value * value, value);
   }
+
+  @Override
+  public double getTotalCurrentDrawn() {
+    return PowerController.getCurrentDrawnFromPDP(fLeftMotorPort, fRightMotorPort, bLeftMotorPort, bRightMotorPort);
```

Comment:
Perhaps this can instead be a `getPDPPorts` method and have `PowerController` call `getCurrentDrawn` itself.